.github/ISSUE_TEMPLATE/maintainer-nomination.yaml

```
@@ -52,7 +52,7 @@ body:
       label: Nomination Review Checklist
       description: Leave this empty. This checklist will be filled in by a sig-docs-community member who is reviewing the nomination.
       options:
-        - label: Has been a Reviewer for the SIG ([docs-reviewer](https://github.com/orgs/o3de/teams/docs-reviewers), community-reviewer, or website-reviewer) for 2+ months.
+        - label: Has been a Reviewer for the SIG ([docs-reviewer](https://github.com/orgs/o3de/teams/docs-reviewers), [community-specialist](https://github.com/orgs/o3de/teams/community-specialist), or website-reviewer) for 2+ months.
```

Is this the right link for website reviewer?
```suggestion
        - label: Has been a Reviewer for the SIG ([docs-reviewer](https://github.com/orgs/o3de/teams/docs-reviewers), [community-specialist](https://github.com/orgs/o3de/teams/community-specialist), or [website-reviewer](https://github.com/orgs/o3de/teams/sig-docs-community-site-reviewers)) for 2+ months.
```